include/command.h

```diff
@@ -200,6 +200,29 @@ class Command
         void update_key(std::string old_key,
                         std::string new_key);
 
+        /*!
+        *   \brief Set address and port for command
+        *          to be executed on
+        *   \param address Address of database
+        *   \param port Port of database
+        */
+        void set_exec_address_port(std::string address,
+                                   uint64_t port);
+
+        /*!
+        *   \brief Get address that command will be
+        *          to be executed on
+        *   \return std::string of address
+        */
+        std::string get_address();
+
+        /*!
+        *   \brief Get port that command will be
+        *          to be executed on
```

Comment:
I think we should also specify the behavior of what gets returned in the case an address and port hasn't been set on the ``Command``.